calibrate.py

```diff
@@ -58,13 +92,29 @@ def write_header_file(board_number, calibration):
     os.rename(fname+'.tmp', fname)
     print('calibration data writed to {}'.format(fname))
 
+
+def write_pickled_data(board_number, calibration):
+    fname = './board_calibration_data/pickled_board_{}'.format(board_number)
+    fpickle = open(fname, 'wb')
+    picle.dump(calibration ,fpickle)
+    
+
+def read_pickled_data(board_number):
+    fname = './board_calibration_data/pickled_board_{}'.format(board_number)
+    mode = 'rb' is os.path.exist(fname) else 'wb'
```

Comment:
Why opening as write when the file does not exist? In this method you are only reading the file.